pubspec.yaml

```
@@ -1,6 +1,7 @@
 name: source_span
 version: 1.9.2-dev
-description: A library for identifying source spans and locations.
+description: >
```

```suggestion
description: >-
```

https://yaml-multiline.info/